packages/docs/src/lang/en/components/Selects.json

```diff
@@ -53,7 +53,7 @@
     "dark": "Components.Inputs.props.dark",
     "debounceSearch": "Debounces the search input value being emitted",
     "deletableChips": "Adds a remove icon to selected chips",
-    "dense": "Lowers max height of list items",
+    "dense": "Reduces the field height and lowers max height of list items",
```

Comment:
should be removed and added to `v-input`'s description, or removed and added to the example description if its something specific to this component.